internal/goldendataset/metric_gen.go

```diff
@@ -0,0 +1,201 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package goldendataset
+
+import (
+	"fmt"
+
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.opentelemetry.io/collector/internal/data"
+)
+
+// Simple utilities for generating metrics for testing
+
+type MetricCfg struct {
```

Comment:
Please add a comment to all public structs/func/members :)

Reply:
Done

Reply:
Probably you forgot to update the PR?

Reply:
Sorry, I should have waited to respond until the PR was ready. It's ready now and pushed.